motulator/control/grid_following.py

```diff
@@ -165,17 +159,19 @@ def __call__(self, mdl):
         else:
             p_g_ref = self.p_g_ref(self.t)
             q_g_ref = self.q_g_ref(self.t)
-
-        
+      
         # Generate the current references
         i_c_ref = self.current_ref_calc.output(p_g_ref, q_g_ref)
-        
+                
         # Use of PLL to bring ugq to zero
-        u_g_q, abs_u_g, theta_pll = self.pll.output(u_g_abc)
+        u_g_q, abs_u_g, w_pll, theta_pll = self.pll.output(u_pcc_abc)
 
         #Transform the measured current in dq frame
         i_c = np.exp(-1j*theta_pll)*abc2complex(i_c_abc)
```

Comment:
Is this theta_pll used in transformations now one step ahead of current and voltage samples to be transformed, e.g. tha angle given by pll.output is theta_pll(k+1) and the angle at time k would be in theta_p stored in the state?